Source/Orts.Formats.OR/WeatherFile.cs

```diff
@@ -253,9 +262,9 @@ public void Save(BinaryWriter outf)
     // fog
     public class WeatherSettingFog : WeatherSetting
     {
-        public float FogVisibilityM = 20000;                           // required fog density - range 0 - 20000
-        public float FogSetTimeS = 300;                                // required rate for fog setting - range 300 - 3600
-        public float FogLiftTimeS = 300;                               // required rate for fog lifting - range 300 - 3600 - required visibility is taken from next weather
+        public float FogVisibility = 1000;                             // required fog density - range 0 - 1000
+        public float FogSetTime = 300;                                 // required rate for fog setting - range 300 - 3600
+        public float FogLiftTime = 300;                                // required rate for fog lifting - range 300 - 3600 - required visibility is taken from next weather
```

Comment:
What units are these values, if not meters and seconds?

Reply:
These are meters and seconds, but the fog variables were the only ones which had this reflected in the names.
Out of consistency, it should either be all or nothing, and 'nothing' led to fewer changes.

Reply:
We have a requirement that [all fields, etc. containing real world measurements must be suffixed with the unit](https://github.com/openrails/openrails/blob/master/CONTRIBUTING.md#physics-requirements).

I would like to set a good example here, and renaming each field is only an F2 away in Visual Studio, so please could you restore these unit suffixes. If consistency dictates adding them to other, related fields, so be it - I would rather we ended up with more correct unit suffixes than less. 🙂

Reply:
Set the suffixes for all relevant variables in the main class definitions.

Reply:
Thanks!